lib/matplotlib/axes/_axes.py

```diff
@@ -3820,6 +3825,10 @@ def bxp(self, bxpstats, positions=None, widths=None, vert=True,
           The widths of the boxes.  The default is
           ``clip(0.15*(distance between extreme positions), 0.15, 0.5)``.
 
+        capwidths : array-like, default: None
+          Either a scalar or a vector and sets the width of each cap.
```

Comment:
Can you make clearer that this is in units of box width?

Reply:
It has the same type requirements as `widths`. I fixed the descrption to match that of `widths`.